azure-pipelines.yml

```diff
@@ -23,65 +23,59 @@ variables:
   - group: dev-secrets
 
 steps:
-- script: |
-    domain=`echo "$(PBURL)" | awk -F[/:] '{print $4}'`
-    if [[ "$(PBURL)" == *"-dev.buyingcatalogue.digital.nhs.uk"* ]]
-    then
-      ip="51.11.46.27"
-    elif [[ "$(PBURL)" == *"private-test.buyingcatalogue.digital.nhs.uk"* ]]
-    then
-      ip="51.132.54.44"
-    elif [[ "$(PBURL)" == *"public-test.buyingcatalogue.digital.nhs.uk"* ]]
-    then
-      ip="51.132.53.156"
-    fi
-    if [[ -n "$ip" ]]
-    then
-      echo "$ip        $domain" | sudo tee -a /etc/hosts
-      cat domain-template.yaml | sed "s/DOMAIN/$domain/g;s/IP/$ip/g" > domain.yml
-    else
-      touch domain.yml
-    fi
-    curl -k -i $(PBURL) --fail --connect-timeout 30
-    cat domain.yml
-    cat /etc/hosts
-  displayName: 'Resolve domain'
-- task: UseDotNet@2
-  displayName: 'Use DotNet Core 3.1'
+- task: UseGitVersion@5
+  name: gitversion
+  displayName: Work out version
   inputs:
-    version: '3.1.101'
```

Comment:
@PhysicsJon are we specifying the dotnet version in the solution?

Reply:
Nope, we should be grabbing latest (the specflow fix is in the variables set when building the image)

Reply:
Put the task back, we're now grabbing latest patch version from 3.1.x